src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html

```diff
@@ -21,24 +21,38 @@
     <div data-ng-if="$ctrl.currentLineDiagnostic.hasQuestionToAnswer('modemIsSynchronized')">
         <form class="form-group"
               novalidate>
-            <oui-field data-label="{{:: 'tools_lineDiagnostics_detection_step_modem_question' | translate }}"
-                       data-label-popover="{{:: 'tools_lineDiagnostics_detection_step_modem_question_help' | translate }}">
-                <oui-radio-toggle-group class="d-inline-block"
-                                        data-model="$ctrl.currentLineDiagnostic.data.answers.modemIsSynchronized"
-                                        data-on-change="$ctrl.answerQuestion('modemIsSynchronized', modelValue)"
-                                        data-ng-hide="$ctrl.loading">
-                    <oui-radio data-value="true">
-                        <span data-translate="tools_lineDiagnostics_detail_yes"></span>
-                    </oui-radio>
-                    <oui-radio data-value="false">
-                        <span data-translate="tools_lineDiagnostics_detail_no"></span>
-                    </oui-radio>
-                </oui-radio-toggle-group>
-                <p data-ng-if="$ctrl.loading">
-                    <oui-spinner data-size="s"></oui-spinner>
-                    <span class="ml-2" data-translate="tools_lineDiagnostics_please_wait"></span>
-                </p>
-            </oui-field>
+            <label class="oui-label">
```

Comment:
There are more than one `labelable` element in this `label`, please use `for="id"` instead

Reply:
`id="radioLabel"`